### PR TITLE
navbar, footer, banner, searchbar, favicon, bootstrap

### PR DESCRIPTION
**Pull request by cypradoux**
Creation of the navbar, with two buttons which returns to the sign up and subsequently the creation of announcements and bicycles. We still had some work to do for navbar's responsivity.
The logo in the middle with a place on the home page and a login button, with a register link on the left which becomes Profile once connected.
Creation of the banner with the search bar (only front, not functionnal) and the footer (no link for the moment). The backgroung image will be change for google maps.
Add bootstrap theme with our colors
Add favicon
Add image for banner (temporaly) in file style.css

Preview index with navbar : 
index
Preview footer :
footer
Preview favicon :
favicon

### CODE REVIEW

**Comment by masechkacat** [comment]
Completed the review of the recent style additions. A few things need clarification and adjustment:

Readme Update: Please include layouts or at least screenshots of the relevant pages to better understand the applied styles.

Bootstrap.css File: The file is named 'bootstrap.css', but it appears to be customized. Can we rename it to reflect its purpose more accurately?

Forms Redundancy: Noticed that new forms have been added, but some seem to duplicate existing ones already connected to the path. Can you please clarify the rationale behind this?

Let's ensure our app for the naming are as clear and functional as the code itself. Looking forward to your input on these points.

File: app/views/shared/_new_bicycle_form.html.erb
Comment (masechkacat) [inline app/views/shared/_new_bicycle_form.html.erb:1]: preciser a quoi ca sert

File: app/views/shared/_register_form.html.erb
Comment (masechkacat) [inline app/views/shared/_register_form.html.erb:1]: preciser a quoi ca sert

File: app/views/shared/_session_form.html.erb
Comment (masechkacat) [inline app/views/shared/_session_form.html.erb:1]: preciser a quoi ca sert

**Review by masechkacat — DISMISSED** [review]
de ma part y a pas de conflit sur ma partie d'app - merge

File: app/views/shared/_new_bicycle_form.html.erb
Comment (cypradoux) [reply app/views/shared/_new_bicycle_form.html.erb:1]: useless

File: app/views/shared/_register_form.html.erb
Comment (cypradoux) [reply app/views/shared/_register_form.html.erb:1]: pour appeler avec render ensuite dans view register#new

File: app/views/shared/_session_form.html.erb
Comment (cypradoux) [reply app/views/shared/_session_form.html.erb:1]: pour appeler avec render ensuite dans view session#new

File: app/assets/stylesheets/bootstrap.css
[the remark this answers is not in the document]
Comment (cypradoux) [reply app/assets/stylesheets/bootstrap.css:1]: o

**Comment by ikramiste** [comment]
- just a reminder that all pictures will need an alt attribute (like you did for logo) when it comes to the project requirements
- avoid uploading images from CSS in the future, i would recommend to the use of the Ruby on Rails image_tag instead with asset_path or HTML so we make sure alt are present

Other than that all good 💯 

No conflicts on my end either OK for merge - @JerkStanton godd for you too ?

**Review by JerkStanton — APPROVED** [review]
Okay then.

**Pull request merged**